app/models/manageiq/providers/kubernetes/container_manager/metrics_capture/hawkular_client_mixin.rb

```diff
@@ -25,7 +25,7 @@ def hawkular_options
      :verify_ssl => @ext_management_system.verify_ssl_mode}
   end
 
-  def status
-    @client.http_get('/status')
+  def availability(id = 'all', end_time = Time.now.to_i * 1_000, start_time = end_time - 20_000)
```

Comment:
@yaacov what about UTC or not synchronized clocks? IIRC you can have queries where the time is evaluated on the hawkular side, e.g.: something like `start=-20s&end=now`

cc @pilhuhn